src/tests/utils/Load.test.js

```diff
@@ -1,9 +1,11 @@
 import { unzipDependencies, loadDependenciesInStorage, loadAsFHIRDefs } from '../../utils/Load';
-import { FHIRDefinitions } from 'fsh-sushi/dist/fhirdefs/FHIRDefinitions';
+import { fhirdefs } from 'fsh-sushi';
 import tarStream from 'tar-stream';
 import http from 'http';
 import 'fake-indexeddb/auto';
 
+const FHIRDefinitions = fhirdefs.FHIRDefinitions;
```

Comment:
Could you do this more simply by changing the import to:
```
import { FHIRDefinitions } from 'fsh-sushi/fhirdefs';
```
I think that should work the same way, and as far as I know that's the more conventional way to do this. And I think that is what we do in GoFSH as well.

Reply:
I notice this in some other places as well, so same question goes for those other places.

Reply:
No, IIRC, you actually _can't_ do that.  Because `fsh-sushi/fhirdefs` is not an actual file in the package.  `fhirdefs` is an exported variable from the package's `index.js`.

Reply:
That said, I'd love to be wrong about this.

Reply:
You are not wrong about this, @jafeltra, please ignore my kinda annoying question.

Reply:
Okay I shall ignore. I believe Chris is correct at to why we can't do it that way. I tried to stick with the way GoFSH does it, although GoFSH does not reassign the imports to variables, it just uses things like `fhirdefs.FHIRDefinitions` directly, so I could swap what I did here to mirror that more closely, if preferred.